_docs/individual-payments.md

```diff
@@ -400,7 +400,7 @@ For mobile payouts to Senegal and Ivory Coast, please use:
   "last_name": "Last",
   "mobile_provider": "orange", // lowercase, see provider values below
   "phone_number": "774044436", // mobile number in local country format
-  "country": "SN" // "SN" for Senegal (default), "CI" for Ivory Coast
+  "country": "SN" // "SN" for Senegal, "CI" for Ivory Coast, "BF" for Burkina Faso, "ML" for Mali
```

Comment:
As discussed in standup, we should add a mandatory `transfer_reason` attribute if the country is Mali - `ML`:
```suggestion
  "country": "SN" // "SN" for Senegal, "CI" for Ivory Coast, "BF" for Burkina Faso, "ML" for Mali
  "transfer_reason": "personal_account" // mandatory for Mali payouts, optional otherwise
```

Reply:
**P.S.:** Please double-check the actual message they want to display with Product

Reply:
I'm waiting for feedback for the actual message from the product team.